server/src/api/discounts/discounts.router.ts

```diff
@@ -86,7 +86,11 @@ discountsRouter.put('/:id', checkJwt, checkScopes, async(
 ) => {
     try {
       const oldCode = await alterDiscountCode(req.params.id);
```

Comment:
We should refactor `oldCode` to `resp` everywhere

Reply:
Fixed